### PR TITLE
Revive benchmarks

### PR DESCRIPTION
**Pull request by greshilov**
Last significant change in this repo was made almost 3 years ago. So much has changed since then in `aiohttp`!

I strongly believe, that performance benchmark is a very important part for any http server project. That's why I've updated dependencies and ran benchmark using two separate amazon instances (`t2.medium` to be precise).

Results are presented in this PR.

-----
[View rendered README.md](https://github.com/greshilov/aiohttp-benchmarks/blob/revive-benchmarks/README.md)

### CODE REVIEW

**Comment by greshilov** [comment]
@samuelcolvin PTAL

File: README.md [hunk 1]
```diff
@@ -31,507 +30,233 @@ scp -i benchmarks.pem -r remote-setup $SSH_ADDRESS:
...
-/queries/{queries}      3.6      raw       20      256       2,009     2,022 |  1%     2,252 | 12% 
-
-/updates/{queries}      3.6      orm        5      256         829       830 |  0%       858 |  3% 
-/updates/{queries}      3.6      orm       10      256         405       451 | 11%       450 | 11% 
-/updates/{queries}      3.6      orm       20      256         223       229 |  3%       198 |-11% 
-/updates/{queries}      3.6      raw        5      256       2,800     2,763 | -1%     3,076 | 10% 
-/updates/{queries}      3.6      raw       10      256       1,744     1,724 | -1%     1,869 |  7% 
-/updates/{queries}      3.6      raw       20      256         959     1,000 |  4%     1,053 | 10%
+               URL   python       DB  queries     Conc        3.6             3.7
+             /json      3.7        -        -       32      7,140     6,723 | -6%
```
Comment (Dreamsorcerer) [inline README.md:49]: If I understand this correctly, these numbers are the number of requests per second or similar? Therefore higher number is better? Can we add this information just before the table, so it's clearer what we're looking at?

File: run.py [hunk 2]
```diff
@@ -38,9 +38,8 @@
 RUN_COMMAND = 'sudo ~/env3{py_v}/bin/python serve.py'
 
 AIOH_VERSIONS = {
-    '2.0': 'aiohttp>=2.0,<2.1',
-    '2.2': 'aiohttp>=2.2,<2.3',
-    '2.3a': 'https://github.com/aio-libs/aiohttp/archive/a60bfb4b4b3eb4d530c2871080201d700c6ed0f9.zip',
```
Comment (Dreamsorcerer) [inline run.py:43]: Can we use these github links like before to include results for the in-development 3.8 branch and also master (for version 4)?

**Review by Dreamsorcerer — CHANGES_REQUESTED** [review]

**Comment by samuelcolvin** [comment]
Are you sure we wouldn't be better off just contributing to the [techempower framework benchmarks](https://www.techempower.com/benchmarks/)?

Keeping this up-to-date seems like a lot of work, is it really worth the effort?

**Comment by greshilov** [comment]
> Are you sure we wouldn't be better off just contributing to the [techempower framework benchmarks](https://www.techempower.com/benchmarks/)?
> 
> Keeping this up-to-date seems like a lot of work, is it really worth the effort?

I thought `techempower` is more about comparing different frameworks between each other.   
These benchmarks allow us to compare different versions of aiohttp and spot changes with performance degradation.

File: README.md (hunk 1 above)
Comment (greshilov) [reply README.md:49]: Good point, fixed!

File: run.py (hunk 2 above)
Comment (greshilov) [reply run.py:43]: Unfortunately, it's not possible to correctly install pip package from github via archive if package has cython extensions.

See more [here](https://github.com/aio-libs/aiohttp/issues/3907).

**Comment by samuelcolvin** [comment]
> > Are you sure we wouldn't be better off just contributing to the [techempower framework benchmarks](https://www.techempower.com/benchmarks/)?
> > Keeping this up-to-date seems like a lot of work, is it really worth the effort?
> 
> I thought `techempower` is more about comparing different frameworks between each other.
> These benchmarks allow us to compare different versions of aiohttp and spot changes with performance degradation.

Makes sense.

**Pull request merged**